client/pkg/transport/listener_tls.go

```diff
@@ -125,6 +125,9 @@ func (l *tlsListener) acceptLoop() {
 	for {
 		conn, err := l.Listener.Accept()
 		if err != nil {
+			if ne, ok := err.(net.Error); ok && ne.Temporary() {
+				continue
+			}
```

Comment:
The `Temporary` has already been deprecated. It seems that you should use `Timeout()` instead

https://github.com/golang/go/blob/53af292aed21c3f6ea75d16e5b25f660b2c028fb/src/net/net.go#L424-L433